tests/testplan.h

```diff
@@ -6,13 +6,15 @@
 class TestPlan : public QObject {
 	Q_OBJECT
 private slots:
-	void testImperial();
 	void testMetric();
+	void testImperial();
```

Comment:
Good question but simple answer:
I prefered to have the single tests running in the same order as there are written down in the testplan.cpp. This was not the case before my changes.
Technically this is for sure not needed but having the results in the same order as in the source is less confusing. Some test dives are very simmilar to another...